features/rating-auth.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Rating Authentication
+  Scenario: Cannot rate a church without permission
+    Given I am a user with permission: ''
```

Comment:
I wonder if this line would read better as: `I am a user without any permission`  or `I am a user with permission: 'not_rate_questions'`.